src/WebJobs.Script.Grpc/Server/GrpcServer.cs

```diff
@@ -15,9 +15,9 @@ public class GrpcServer : IRpcServer, IDisposable
         private Server _server;
         private bool _disposed = false;
 
-        public GrpcServer(FunctionRpc.FunctionRpcBase serviceImpl)
+        public GrpcServer(FunctionRpc.FunctionRpcBase serviceImpl, int grpcMaxMessageLength = 32 * 1024 * 1024)
```

Comment:
Its not great that the constant value is duplicated here... Any reason why this param has to be optional?